lib/models/methods.js

```diff
@@ -53,42 +54,42 @@ MethodsModel.prototype._getMetrics = function(timestamp, method) {
       })
     };
 
-    METHOD_METRICS_FIELDS.forEach(function(field) {
+    METHOD_METRICS_FIELDS.forEach(function (field) {
       methods[method][field] = 0;
     });
   }
 
   return this.methodMetricsByMinute[dateId].methods[method];
 };
 
-MethodsModel.prototype.setStartTime = function(timestamp) {
+MethodsModel.prototype.setStartTime = function (timestamp) {
   this.metricsByMinute[dateId].startTime = timestamp;
```

Comment:
`dateId` is undefined. Is it OK?

Reply:
We probably can remove it in that case. It looks like its been broken and unused since it was added.